jw7-plugin-sdk/plugins/repeat/repeat.js

```diff
@@ -0,0 +1,48 @@
+(function (jwplayer) {
+    var template = function (player) {
+        var i = 0;
+        var state = "";
+        player.onReady(setup);
+        function setup() {
+            addIconCss();
```

Comment:
See below comment - remove `addIconCss` from setup and run in plugin module function (template function).